### ISSUE TITLE
Sub-Menu Link Incorrect

### ISSUE THREAD

**Issue by dfhawthorne** [dfhawthorne/dfhawthorne.github.io#64]
The following link, from `docs/home/12c-ocp-upgrade/section-1-new-features-of-oracle-database-12c/emergency-monitoring-real-time-addm-compare-period-addm-and-active-session-history-ash-analytics/diagnose-performance-issues-.html`, refers back to itself:

```html
<a href="home/12c-ocp-upgrade/section-1-new-features-of-oracle-database-12c/emergency-monitoring-real-time-addm-compare-period-addm-and-active-session-history-ash-analytics/diagnose-performance-issues-.html">
  Diagnose performance issues using ASH enhancements
</a>
```

**Comment by dfhawthorne** [comment]
Another occurrence is:

```html
<a href="home/12c-ocp-upgrade/section-1-new-features-of-oracle-database-12c/emergency-monitoring-real-time-addm-compare-period-addm-and-active-session-history-ash-analytics/perform-emergency-monitoring.html">
  Perform emergency monitoring and real-time ADDM
</a>
```

**Comment by dfhawthorne** [comment]
This was caused by commit [2ea7fd0](https://github.com/dfhawthorne/dfhawthorne.github.io/commit/2ea7fd0a90bae8b381b0fd4b6cfe05c044b39228)

**Comment by dfhawthorne** [comment]
This was fixed by issue #65 .
